### PR TITLE
Disables mgmt build on sdk/ folder and disables old azure-sdk-for-net…

### PR DESCRIPTION
**Pull request by mitchdenny**
This PR finalizes the transition to the shorter running PR validation pipelines that have been setup. Basically once this merges people who are working exclusively in the ```sdk/``` folder won't need to worry about waiting for the ```azure-sdk-for-net - mgmt``` or ```azure-sdk-for-net - client``` pipelines to run.

### CODE REVIEW

File: .azure-pipelines/mgmt.yml
Comment (weshaggard) [inline .azure-pipelines/mgmt.yml]: This is only going to help temporarily as @chidozieononiwu is soon going to move the mgmt libraries under the sdk folder as well.

File: .azure-pipelines/client.yml
Comment (weshaggard) [inline .azure-pipelines/client.yml]: We should be able to completely delete this file now. We can use the eng/pipelines/client.yml for our full builds and release pipelines.

**Comment by mitchdenny** [comment]
I've disabled the CI and PR triggers in the defintiion in Azure DevOps so it won't fire anymore and I've removed the ```client.yml```l file in this PR. Once its merged I'll hold off deleting the old pipeline definition for a while since once we delete it all the history is gone. But we can clean it up in a week or so (unless you don't think that there is any value in keeping it around for a bit longer).

File: .azure-pipelines/mgmt.yml
Comment (mitchdenny) [reply .azure-pipelines/mgmt.yml]: Yep. I'm thinking when we do this we might get really specific with the path filters?

**Comment by weshaggard** [comment]
> But we can clean it up in a week or so (unless you don't think that there is any value in keeping it around for a bit longer).

For the one in the public team there isn't any real value in keeping it around. We still have the equivalent "net - client" in internal.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**
